modules/tensor_mechanics/include/materials/DamageBase.h

```diff
@@ -49,6 +49,12 @@ class DamageBase : public Material
    */
   virtual void updateJacobianMultForDamage(RankFourTensor & jacobian_mult) = 0;
 
+  /**
+   * Compute the limiting value of the time step for this material
+   * @return Limiting time step
+   */
+  virtual Real computeTimeStepLimit() = 0;
```

Comment:
I would prefer to follow the pattern in StressUpdateBase where we have this implemented in the base class, and it returns `std::numeric_limits<Real>::max()`. That would avoid the issue of temporarily breaking BlackBear.